cherrypy/test/test_tools.py

```diff
@@ -343,6 +347,7 @@ def testGuaranteedHooks(self):
         expected_msg = tmpl.format(attr='items')
         self.assertInBody(expected_msg)
 
+    @pytest.mark.skipif(ci_on_macOS, reason="continuous integration on macOS fails")
```

Comment:
The problem here was unrelated to the OS but to a patch in CPython. I've fixed it on master via https://github.com/cherrypy/cherrypy/pull/1879 (see https://github.com/cherrypy/cherrypy/issues/1849 / https://bugs.python.org/issue39389 / https://docs.python.org/3.7/whatsnew/changelog.html#python-3-7-7-final).

But not that it's fixed, this mark shouldn't be necessary. Also, even if it was necessary, it'd be xfail, not skipif — this one doesn't crash the test runner.

Reply:
Removed because 1879 should fix this.